exporter/googlecloudpubsubexporter/go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter
 
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.11
```

Comment:
Is this `toolchain` change something we want?

Reply:
@dashpole Could you take a look?

Reply:
should no longer be an issue now that we use Go 1.23